rust/rsc/src/rsc/blob_store_service.rs

```diff
@@ -20,3 +21,47 @@ pub async fn fetch_local_blob_stores(
 ) -> Result<Vec<local_blob_store::Model>, DbErr> {
     LocalBlobStore::find().all(db).await
 }
+
+pub async fn fetch_unreferenced_blobs(
```

Comment:
Can we get a comment describing what this does because the TTL aspect isn't clear from the function name.